### DISCUSSION TITLE
Any hope for "Open in Notion" link to come back?

### DISCUSSION THREAD

**Discussion by 22-modifier** [2anki/server#1108] [Q&A]
I found this to be clutch when making cards as something that seems good in Notion may not actually make for a great card in Anki. Having the direct link back made it really easy to edit. Then I could just re-upload via HTML uploader and card would update. It was amazing!

**Comment by aalemayhu** [comment]
I will bring it back but the focus currently is on stabilizing the features we have. Thanks 👍🏾

**Comment by aalemayhu** [comment]
Follow this issue for future updates https://github.com/2anki/2anki.net/issues/1124
